apps/wallet/src/shared/messaging/messages/payloads/keyring/index.ts

```diff
@@ -25,12 +22,12 @@ type MethodToPayloads = {
     };
     walletStatusUpdate: {
         args: never;
-        return: Partial<{
+        return: {
             isLocked: boolean;
             isInitialized: boolean;
-            // we can replace keypair (once we stop signing from the UI) with the account address
-            activeAccount: ExportedKeypair;
-        }>;
+            accounts: ReturnType<Account['toJSON']>[];
```

Comment:
Probably should make the `toJSON` return structure a type or interface so that you can use it without needing to extract the returntype.